src/Matches/Matches.js

```diff
@@ -51,7 +52,43 @@ class Matches extends React.Component {
             alert("Something went wrong");
             console.error(error)
         });
-  }
+    }
+    unmatchUser(event) {
+        const userSelected = event.target.value.split("|");
+        const myRequest = new Request('http://127.0.0.1:5000/unmatch', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify({
+                "userId": this.state.userId,
+                "friendId": userSelected[0],
+            }),
+        });
+        fetch(myRequest)
+        .then(response => response.json())
+        .then(res => 
+            (res.userIds && res.userIds.length != 0) ?
+                this.setState({
+                    name: res.currentName,
+                    matchesExist: "exists", 
+                    userIds: res.userIds,
+                    notMessagedUserIds: res.notMessagedUserIds,
+                    notMessagedUserNames: res.notMessagedUserNames,
+                    messagedUserIds: res.messagedUserIds,
+                    messagedUserNames: res.messagedUserNames,
+                    messageIds: res.messageIds,
+                    messageSender: res.messageSender,
+                    messageContent: res.messageContent,
+                    timeStamp: res.timeStamp,
+                })
+                :
+                this.setState({
+                    matchesExist: "not exists"
```

Comment:
Wouldn't it be better for this variable to be a boolean?

Reply:
technically this could be either empty, "exist", or "not exists", so there are 3 possible states